charts/backingservices/charts/srs/README.md

```diff
@@ -18,7 +18,8 @@ The service deployment provisions runtime service pods along with a dependency o
 | Pega Infinity version | SRS version | ElasticSearch version | Description                                                                                                                                                                                                                                                                                                           |
 |-----------------------|-------------|-----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | < 8.6                 | NA          | NA                    | SRS can be used with Pega Infinity 8.6 and later                                                                                                                                                                                                                                                                      |
-| \>= 8.6               | \>=1.17.10  | 7.10.2 or 7.16.3      | Pega Infinity 8.6 and later supports using a Pega-provided platform-services/search-n-reporting-service Docker Image. While all SRS Docker image versions starting at 1.12.0 are certified against Elasticsearch versions 7.10.2 and 7.16.3, Pega recommends using the latest available SRS image - 1.17.10 or later. |
+| \>= 8.6 & \<= 8.8.0              | \>= 1.17.10 & \< 1.20.7  | 7.10.2 or 7.16.3      | Pega Infinity 8.6 and later upto 8.8.0 supports using a Pega-provided platform-services/search-n-reporting-service Docker image. While all SRS Docker image versions starting at 1.12.0 are certified against Elasticsearch versions 7.10.2 and 7.16.3, Pega recommends using the latest available SRS image - 1.20.7 or later.                                                                                                                                                                                                                                                                     |
+| \>= 8.6              | \>= 1.20.7  | 7.10.2 or 7.16.3 or 7.17.9     | Pega Infinity 8.6 and later supports using a Pega-provided platform-services/search-n-reporting-service Docker Image. Pega Infinity 8.8.1 and later only supports Docker image 1.20.7 or later. While 1.20.7 Docker image is certified againts Elasticsearch version 7.10.2, 7.16.3 and 7.17.9, Pega strongly recommends using the Elasticsearch version 7.17.9.|
```

Comment:
please consider adding a focus on why to use the later version...in the previous paragraph your guidance pushes people to use 1.20.7 or later. So here, we should add more engaging guidance>>
Pega Infinity 8.6 and later supports using the latest Docker Image to use the latest supported Elasticsearch version, 7.17.9, which Pega strongly recommends using; Pega Infinity 8.8.1 and later requires Docker image 1.20.7 or later.

Reply:
Addressed the comment.. @ravitejamekapega